### PR TITLE
Fixed broken link in Introduction section and addressed Github issue #47

### PR DESCRIPTION
**Pull request by rachelsun97**

### CODE REVIEW

File: training/04_editing_eml.Rmd [hunk 1]
```diff
@@ -87,3 +87,22 @@ qa_package(mn, resource_map_pid)
...
+
+One of the biggest dilemmas when handling data packages is deciding when it's appropriate to touch a PI's data. As a rule of thumb, data entries themselves should always be fixed. Our goal is to clean-up the files, not to revise the data. 
+
+Data file conversions are generally alright. Conversions are needed when the existing file format may become unavailable in the future. In the case of Excel sheets, since they are stored in a proprietary software, the files may eventually become inaccessible. Storing them in a non-proprietary software, such as a csv or text file, prevents it from becoming extinct. 
+
+Data files should also be revised if there are formatting issues. For instance, column headers should be fixed if there exists special characters, spaces, etc. Similarly, if there exists blank entries without missing value codes, they should be filled in with `NA` values with their respective explanations. These changes should be documented somewhere in the description or in an attribute definition.
+
+Finally, altered data files should be linked to the originals through provenance.
```
Comment (dmullen17) [inline training/04_editing_eml.Rmd]: You should add here that if possible we want to include a processing script that does any transformations / conversions.

File: training/04_editing_eml.Rmd [hunk 2]
```diff
@@ -87,3 +87,22 @@ qa_package(mn, resource_map_pid)
...
+
+One of the biggest dilemmas when handling data packages is deciding when it's appropriate to touch a PI's data. As a rule of thumb, data entries themselves should always be fixed. Our goal is to clean-up the files, not to revise the data. 
+
+Data file conversions are generally alright. Conversions are needed when the existing file format may become unavailable in the future. In the case of Excel sheets, since they are stored in a proprietary software, the files may eventually become inaccessible. Storing them in a non-proprietary software, such as a csv or text file, prevents it from becoming extinct. 
+
+Data files should also be revised if there are formatting issues. For instance, column headers should be fixed if there exists special characters, spaces, etc. Similarly, if there exists blank entries without missing value codes, they should be filled in with `NA` values with their respective explanations. These changes should be documented somewhere in the description or in an attribute definition.
```
Comment (dmullen17) [inline training/04_editing_eml.Rmd]: I'm not totally sure we want to be doing this.  @jeanetteclark what do you think?

File: training/04_editing_eml.Rmd [hunk 3]
```diff
@@ -87,3 +87,22 @@ qa_package(mn, resource_map_pid)
...
+
+Finally, altered data files should be linked to the originals through provenance. 
+
+When it comes to methods, the information is sufficient when a reasonable scientist can interpret the data and replicate the study without consulting the PI nor any other resources. Enough information should be provided such that users can understand how the data were collected, how to interpret the values, and potentially how to use the data in the case of specialized files. The abstract should provide an overview of the scientific project, how the data package fits into the larger project, a synopsis of the experimental or sampling design, and a summary of the data contents. 
+
+If either of these sections are insufficient, the PI should be contacted for revisions. In the case that the project is funded by an NSF grant, a missing/insufficient abstract can be replaced by the abstract provided on the NSF funding page.
```
Comment (dmullen17) [inline training/04_editing_eml.Rmd]: I would change this wording a little bit "a missing/insufficient abstract can be replaced by the abstract provided on the NSF funding page."

```
You can recommend to the PI that they supplement their missing/insufficient abstract with language included in their project award abstract. 
```

File: training/04_editing_eml.Rmd [hunk 4]
```diff
@@ -87,3 +87,22 @@ qa_package(mn, resource_map_pid)
...
+
+An exception to the methods/abstract guidelines would be if the data package has a parent that it will eventually be nested under. If the parent contains a comprehensive abstract and methods, then the individual data packages can have more brief information.
```
Comment (dmullen17) [inline training/04_editing_eml.Rmd]: This is good.  I would add in most cases the parent does not need to contain methods information.

File: training/04_editing_eml.Rmd (hunk 2 above)
Comment (jeanetteclark) [reply training/04_editing_eml.Rmd]: The first paragraph of this is correct, but not the second. I don't think we should be reformatting plain text files unless there is an extremely compelling reason. This is a bit of a diversion from some of our previous workflow policies but I've talked with Matt and Dominic about this a fair bit

File: training/04_editing_eml.Rmd (hunk 4 above)
Comment (jeanetteclark) [reply training/04_editing_eml.Rmd]: Yes, although we should stop offering to nest unless a PI is really insistent about it

**Pull request closed without merging**